packages/@okta/eslint-plugin-okta-ui/lib/rules/no-bare-templates.js

```diff
@@ -0,0 +1,61 @@
+const DEFAULT_MESSAGE = 'Template literal should be precompiled. Wrap template string in a call to hbs(\'template\') or use template syntax: hbs`template`';
+
+function isHandlebarsTemplate(node) {
+  if (node.type === 'Literal' && typeof node.value === 'string') {
+    return node.value.indexOf('{{') >= 0;
+  }
+  if (node.type === 'TemplateLiteral') {
+    node.quasis.forEach(quasi => {
+      let value = quasi.value.cooked;
+      if (value.indexOf('{{') >= 0) {
+        return true;
+      }
+    });
+  }
+  return false;
+}
+
+function isWrapped(node) {
+  if (node.parent.type === 'TaggedTemplateExpression' && node.parent.tag.name === 'hbs') {
+    return true;
+  }
+  if (node.parent.callee && node.parent.callee.name === 'hbs') {
+    return true;
+  }
+  return false;
+}
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Detect use of bare (unwrapped) handlebars template strings',
+      category: 'CSP Issues',
+      recommended: true,
```

Comment:
Would also be nice to provide logic for `--fix` since the change is minimal.

Reply:
Good suggestion. This file is also copied from `okta-ui`, the original source is in a separate PR: https://github.com/okta/okta-ui/pull/2127